harness/dotnet/Temporalio.Features.Harness/Runner.cs

```diff
@@ -152,6 +198,43 @@ public async Task SkipIfAsyncUpdateNotSupportedAsync()
             Client.GetWorkflowHandle("does-not-exist").StartUpdateAsync(
                 "does-not-exist", Array.Empty<object?>()));
 
+    /// <summary>
+    /// Stop the worker.
+    /// </summary>
+    public void StartWorker()
```

Comment:
I wonder if there would be value in a single `RestartWorker` call.

Reply:
Maybe, but, it's a bit less flexible